src/test/scala/mesosphere/marathon/api/validation/V2AppDefinitionValidatorTest.scala

```diff
@@ -23,6 +25,109 @@ class V2AppDefinitionValidatorTest extends MarathonSpec {
     validateJsonSchema(app)
   }
 
+  for (
```

Comment:
Ok. I see your point. Without the programmatic creation we have more code and its more redundant but easier to debug a single test. I changed it.